python/ray/_private/worker.py

```diff
@@ -444,15 +445,9 @@ def __init__(self):
         self.node = None
         self.mode = None
         self.actors = {}
-        # A dictionary that maps from an object ID to a list of tensors.
-        self.in_actor_object_store: Dict[str, List[torch.Tensor]] = {}
-        # A dictionary that maps from an object ref to a tuple of (actor handle, object ref).
-        # The key of the dictionary is an object ref that points to data consisting of tensors.
-        # These tensors are stored in the in-actor object store of the actor referenced by the ActorHandle.
-        # The object ref in the tuple is the object ref of a list of tuples, each containing the shape and dtype of a tensor.
-        self.in_actor_object_refs: Dict[
-            ObjectRef, Tuple["ray.actor.ActorHandle", ObjectRef]
-        ] = {}
+        # GPU object manager to manage GPU object lifecycles, including coordinating out-of-band
+        # tensor transfers between actors, storing and retrieving GPU objects, and garbage collection.
+        self.gpu_object_manager = GPUObjectManager()
```

Comment:
Suggest renaming to `self._gpu_object_manager` and using @property to define a getter method.

Reply:
Updated https://github.com/ray-project/ray/pull/52938/commits/78ea5b5dd5e5c47f363ee70a32af86c8271aaed3